.github/workflows/n8n-dependency-update.yaml

```diff
@@ -21,22 +22,26 @@ jobs:
         with:
           ref: ${{ inputs.pull_request_ref }}
 
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          cache: npm
+          cache: pnpm
+          node-version: ${{ env.NODE_VERSION }}
 
       - name: Install dependencies
-        run: npm ci
+        run: pnpm install --frozen-lockfile
 
       - name: Retrieve target versions
         id: target-versions
         # yamllint disable rule:line-length
         run: |
           set -e
-          n8n_version=$(npm list n8n --json | jq --raw-output --exit-status '.dependencies.n8n.version')
+          n8n_version=$(pnpm list n8n --filter @skriptfabrik/n8n-nodes --json | jq --raw-output --exit-status '.[0].devDependencies.n8n.version')
```

Comment:
What's the benefit of this filter, and why did you not use it in the other places as well?

Reply:
The command `pnpm list n8n` lists all installations of `n8n` in the entire workspace. The result is an array of objects whose name is the name of the package. We only want to get the version of n8n in the root package.

Reply:
Ah thanks, that's better you are right, thanks for the explanation, did not recognize it myself.